insecure-db-2.tf

```diff
@@ -0,0 +1,12 @@
+resource "aws_db_instance" "default2" {
+  allocated_storage    = 10
+  engine               = "mysql"
+  engine_version       = "5.7"
+  instance_class       = "db.t3.micro"
+  name                 = "mydb"
+  username             = "foo"
+  password = "foobarbaz"
+  parameter_group_name = "default.mysql5.7"
+  skip_final_snapshot  = true
+  publicly_accessible = true
```

Comment:
### :warning: Aqua detected misconfiguration in your code  
**Misconfiguration ID:** AVD-AWS-0082   
**Check Name:** A database resource is marked as publicly accessible.   
**Severity:** CRITICAL   
**Message:** Instance is exposed publicly.  
**Resolution:** Set the database to not be publicly accessible  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/misconfig/avd-aws-0082